docs/source/markdown/podman.1.md

```diff
@@ -1,56 +1,52 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for pods, containers and images
+podman - simple management tool for pods, containers and images
 
 ## SYNOPSIS
-**podman** [*options*] *command*
+**podman command** [*optional*] *value*
 
 ## DESCRIPTION
-Podman (Pod Manager) is a fully featured container engine that is a simple daemonless tool.
-Podman provides a Docker-CLI comparable command line that eases the transition from other
-container engines and allows the management of pods, containers and images.  Simply put: `alias docker=podman`.
-Most Podman commands can be run as a regular user, without requiring additional
-privileges.
+The podman command is used to control the Podman container engine using a CLI.  Most commands can be run as a regular user, without requiring additional privileges. Default settings for flags are defined in `containers.conf` which can be found in `/etc/containers`. Most settings for remote connections use the server’s `containers.conf`, except when documented in man pages.
 
-Podman uses Buildah(1) internally to create container images. Both tools share image
-(not container) storage, hence each can use or manipulate images (but not containers)
-created by the other.
 
-Default settings for flags are defined in `containers.conf`. Most settings for
-Remote connections use the server's containers.conf, except when documented in
-man pages.
+## OPTIONS
 
-**podman [GLOBAL OPTIONS]**
+#### **--cgroup-manager**=**_systemd_** | *cgroupfs*
 
-## GLOBAL OPTIONS
+| CGroup Manager| Description|
+| ----------- | ----------- |
+| systemd (Default)| daemon process for managing services|
+| cgroupfs | pseudo-filesystem for the cgroup interface|
```

Comment:
For some reason this translates in the man page as a third empty box.

Reply:
I am not sure, this looks fine on my build
![Screen Shot 2021-06-03 at 12 27 53 PM](https://user-images.githubusercontent.com/65443164/120601144-35e7ee80-c467-11eb-9670-1c81656f4a62.png)

Reply:
Strange, what OS and what version of gomd2man are you using?

Here is what this patch looks like when converted to nroff on Fedora 35.
Check out all of the tables.

[podman.1.txt](https://github.com/containers/podman/files/6590864/podman.1.txt)

Reply:
I am using a Ubuntu 20.04 VM. 
Checkout this gist: https://gist.github.com/infiniteregrets/4a020243a28988411aaa92a91b010719

Works just fine for me. I also made a few changes that fix the formatting issues